feat: add my delegatee to Delegates page

Sekhmet opened the pull request:
### Summary

This PR currently only supports `governor-subgraph` delegation type. `delegate-registry` will be implemented soon (currently DNS issues cause registry API to be down).

Closes: https://github.com/snapshot-labs/workflow/issues/334

### How to test

1. Go to http://localhost:8080/#/sn:0x009fedaf0d7a480d21a27683b0965c0f8ded35b3f1cac39827a25a06a8a682a4/delegates
2. Sign in with Starknet wallet.
3. You see your delegation.
4. Go to Ethereum tab.
5. Connect with Ethereum wallet.
6. You see your delegation.
7. TODO: http://localhost:8080/#/s:cow.eth/delegates

### Screenshots

<img width="741" alt="image" src="https://github.com/user-attachments/assets/417157de-9ab9-4a5e-825a-9249f37b03c7" />

wa0x6e commented:
Should we exclude delegations to the empty address `0x0000000000000000000000000000000000000000` ? Like for http://localhost:8080/#/sn:0x009fedaf0d7a480d21a27683b0965c0f8ded35b3f1cac39827a25a06a8a682a4/delegates

wa0x6e commented:
On http://localhost:8080/#/sn:0x009fedaf0d7a480d21a27683b0965c0f8ded35b3f1cac39827a25a06a8a682a4/delegates

![Screenshot 2025-01-22 at 22 54 06](https://github.com/user-attachments/assets/5337e895-376a-426c-8702-8f1b000a3f55)

You are delegating 3 STRK to someone (yourself)

Shouldn't that delegatee also appear in the list of delegatee below ?

wa0x6e commented:
What's the `0.11%` percentage based on ? From the full list below, 0.11% is more around 70k vCOW than 9

![Screenshot 2025-01-22 at 22 59 21](https://github.com/user-attachments/assets/ad172d44-496d-4ceb-8058-9daae62067f1)

Sekhmet commented:
@wa0x6e it's percentage of your VP relative to total VP this delegatee got.

wa0x6e commented:
Seems like the graphql delegates fetching is broken and always return an empty delegate list for 

https://sekhmet-my-delegatee-preview-snapshot-devnet.on-fleek.app/#/sn:0x009fedaf0d7a480d21a27683b0965c0f8ded35b3f1cac39827a25a06a8a682a4/delegates

It works on prod: https://snapshot.box/#/sn:0x009fedaf0d7a480d21a27683b0965c0f8ded35b3f1cac39827a25a06a8a682a4/delegates

bonustrack commented:
To undelegate with an ERC20 Votes token you need to delegate to `0x0000000000000000000000000000000000000000` instead of self. The option to undelegate should appear even when I delegate to myself.

Sekhmet commented:
Updated, also converted delegatee fetching to use vue-query as previous PR was merged. With this change when you undelegate after modal triggers confirmation data will be refetched (delegatee list and your own delegatee).

bonustrack reviewed (approved):
tACK

Sekhmet commented:
@wa0x6e let me know if you want to re-review or it's okay to dismiss your previous review.

wa0x6e commented:
> @wa0x6e let me know if you want to re-review or it's okay to dismiss your previous review.

Will review one last time in the next hour

wa0x6e reviewed (approved):
tAck
